src/ITensorAutoHOOT/batch_contraction.jl

```diff
@@ -25,15 +27,23 @@ end
 
 NetworkSum() = NetworkSum([])
 
-function Executor(networks::Array)
-  nodes, node_dict = generate_einsum_expr(networks)
-  net_sums = [NetworkSum([go.generate_optimal_tree(n)]) for n in nodes]
+function Executor(networks::Array{Union{Array{ITensor,1}},1})
```

Comment:
Can use `Vector{ITensor}` instead of `Array{ITensor,1}` (it is just an alias for that).